Oware/Program.fs

```diff
@@ -12,7 +12,31 @@ let start position = failwith "Not implemented"
 
 let score board = failwith "Not implemented"
 
-let gameState board = failwith "Not implemented"
+let gameState board = 
+    match South.score >= 25, North.score >= 25, (South.score = 24 && North.score = 24) with 
+    |_, _, true -> "Game ended in a draw"
+    |true, true, _ -> "Game ended in a draw"
+    |true, false, _ -> "South won"
+    |false, true, _ -> "North won"
+    |false, false, _ -> match state.Turn with 
+        |South -> "South's turn"
+        |North -> "North's turn"
+
+let updateConsole () =
+    System.Console.Clear ()
+    System.Console.WriteLine ("\n\t\t---------------------------------------------------------------\n\t\t\t\t\t " + "South's Turn " + "\n\t\t===============================================================")
+    System.Console.WriteLine ("\t\t                            NORTH")
+    System.Console.WriteLine ("\t\t--------------------------POINTS: 0 ----------------------------")
+    System.Console.WriteLine ("\t\t ||  ||                                                 ||  ||")
```

Comment:
im assuming on a later commit this will be dynamic instead of fixed values

Reply:
yes. ofcourse once the parts I'm getting the values from are completed